CONTRIBUTING.md

```diff
@@ -37,381 +48,295 @@ to sign the CLA. Therefore,
-[ROLES](https://github.com/knative/community/blob/master/ROLES.md).
-
-### Member
-
-Established contributor to the Knative docs.
+The
+[Knative Documentation Working Group](https://github.com/knative/community/blob/master/working-groups/WORKING-GROUPS.md#documentation)
+meets biweekly on Tuesdays at 9:30am PT.
+[Click here](https://calendar.google.com/calendar/embed?src=knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com)
+to see the exact dates on the Knative working group calendar.
 
-Members are continuously active contributors in the community. They can have
-issues and PRs assigned to them, might participate in working group meetings,
-and pre-submit tests are automatically run for their PRs. Members are expected
-to remain active contributors to the Knative documentation.
+The Working Group meetings are used to discuss documentation strategy and
+policy; we expect individual PR review to happen mostly asynchronously via
+GitHub, and [Slack](https://slack.knative.dev).
 
-All members are encouraged to help with PR reviews, although each PR must be
-reviewed by an official [Approver](#approver). In their review, members should
-be looking for technical correctness of the documentation, adherence to the
-[style guide](https://developers.google.com/style/), good spelling and grammar
-(writing quality), intuitive organization, and strong documentation usability.
-Members should be proficient in at least one of these review areas.
+# Workflow
 
-### Requirements
+There are roughly two workflows: writing / contributing new content, and
+refining and reviewing existing content.
 
-- Has made multiple contributions to the project or community. Contributions
-  might include, but are not limited to:
+## New Content
 
-  - Authoring and reviewing PRs on GitHub in the Docs or Website repos.
+We expect most new content to be written by the subject matter expert (SME)
+which would be the contributor who actually worked on the feature or example.
 
-  - Filing and commenting on issues on GitHub.
+When writing new content, keep the following in mind:
 
-  - Contributing to working group or community discussions.
+- Write one [type of documentation](#type of documentation) per page.
 
-- Subscribed to
-  [knative-dev@googlegroups.com](https://groups.google.com/forum/#!forum/knative-dev).
+- If possible, follow the style/template of other documents of the same type.
 
-- Actively contributing to 1 or more areas.
+- Focus mostly on technical correctness; structure and language should be
+  roughly correct, but don't need heavy review in this phase.
 
-- Sponsored by 1 approver.
+The goal of adding new content is to get technically correct documentation into
+the repo before it is lost. Tech Writers may provide some quick guidance on
+getting documentation into the correct location, but won't be providing a
+detailed list of items to change.
 
-  - Done by adding GitHub user to Knative organization.
+## Refining and Reviewing Content
```

Comment:
Per my comments above. Who owns and manages this step of the process?

Perhaps a different process is necessary with the writing resources today? Just ideas, but what about:

1. A "PDRD" product document requirement document is filled out - a question/checklist matrix that narrows down what "type of doc" it is (the overhead to read and learn here might work but i doubt our contributors will spend the time reading and learning https://documentation.divio.com) - my alternative would have just enough questions to nail down and answer the usually missing "why", and then direct that SME to the right doc-type template (Richard mentioned they have some we can use).

2. Then, something more fluid should be used to "capture" the rough draft, like some type of collaborative document (Google doc?) where all the churn and commenting can happen

3. Lastly, the PR happens, any other experts can review and revise before the merge happens and the doc lives on (temporarily? but most likely permanently?).

Reply:
Building on this thought. If we have the other WG TLs own the triaging process of all Issues and PRs, they could be the filter for the limited writing resources that we have today and therefore "tagging" a PR or Issue when it is indeed ready (ie. completed a SME tech review) would/might streamline the process (reduce noice and allow writers to focus solely on actionable items).

General summary/process (writers focus on actionable work):

- new content = fill out this form and answer the questions/provide details that a writer needs

- old/existing content = only address PRs or Issues that have been tagged by WG TLs

Reply:
I don't see who would be writing a "Product Requirement Document" of the type that you're imagining. We have _one_ product manager at the moment over ~50 engineers, and no sort of formal process requiring this kind of gate.

I think we probably need a radically different process that PM-rich environments might have; it's not clear that having tech writers gate all the PRs coming in to docs is the best or most sustainable process. Another way to do this would be to put a priority on document quality based on depth/distance from the root, and then use that to focus the tech writing review and improvements.

Reply:
(Re-reading this in the morning, I realize that it wasn't as positive and solution-focused as I'd like to be.)

One problem which I see is that the engineering side of the project hears "our docs are a problem", and would like to help but end up getting bottlenecked/burned out on the current process because we have limited tech writing resources. I'd like to find a creative solution which allows the tech writers to operate more effectively in a "writing / revising" mode for larger chunks of time, and spend less time feeling interrupt-driven trying to help review PRs.

If we'd rather have the pending docs end up in a Google Docs folder for transcription into GitHub by tech writers, I'm happy to try that as long as it's clear that the "new docs process" is not to go into GitHub.

Reply:
I *do* want to have a "New Docs" section in here; I'm happy to try whatever seems best to @RichardJJG and @snneji , who will be doing the work. 😉